### ISSUE TITLE
Bug in docker build (probably due to url shortener feature)

### ISSUE THREAD

**Issue by cervoneluca** [gitcoinco/web#6160]
**Describe the bug**
I'm not able to build the docker. The build stops with this error: 

```shell
Collecting pyshorteners
  Downloading pyshorteners-1.0.0.tar.gz (9.8 kB)
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-5o2gds2v/pyshorteners/setup.py'"'"'; __file__='"'"'/tmp/pip-install-5o2gds2v/pyshorteners/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-5o2gds2v/pyshorteners/pip-egg-info
         cwd: /tmp/pip-install-5o2gds2v/pyshorteners/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-5o2gds2v/pyshorteners/setup.py", line 8, in <module>
        README = r.read()
      File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 21: ordinal not in range(128)
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Service 'web' failed to build: The command '/bin/sh -c pip3 install --upgrade -r test.txt' returned a non-zero code: 1
```
**To Reproduce**

1) Clone the web repository 
2) fetch the upstream 
3) run `docker-compose up --build`

**Screenshots**
<img width="1676" alt="Schermata 2020-03-06 alle 12 17 50" src="https://user-images.githubusercontent.com/3637606/76079232-880c3c80-5fa4-11ea-9012-e8f9234832e6.png">

**Desktop (please complete the following information):**

- OS: macOs 10.15.3 

**Additional context**

I think that the issue can be related to works made for this pull request  (or, anyway, to works related to url shorteners)  https://github.com/gitcoinco/web/pull/6152

**Comment by owocki** [comment]
this is fixed in ebdb2201352d3956bd60fdb00025600528ef85f9

**Comment by cervoneluca** [comment]
ok solved! I'll close this issue. Anyway, if anyone else have the same problem, refer to https://gitcoin.co/townsquare?tab=activity:158890 

The solution is in comments of the above thread.
